package.json

```diff
@@ -11,6 +11,7 @@
     "src/**/*.vue"
   ],
   "scripts": {
+    "yalc": "yarn build && yarn build:ssr && yarn build:es && yarn build:unpkg && yalc publish",
```

Comment:
For build process we should keep it exactly the same as in the source repository